src/Networks/Rutanio/Rutanio/Networks/RutanioMain.cs

```diff
@@ -0,0 +1,285 @@
+                [RutanioBIP9Deployments.CSV] = new BIP9DeploymentsParameters("CSV", 0, BIP9DeploymentsParameters.AlwaysActive, 999999999, BIP9DeploymentsParameters.DefaultTestnetThreshold),
+                [RutanioBIP9Deployments.Segwit] = new BIP9DeploymentsParameters("Segwit", 1, new DateTime(2020, 6, 1, 0, 0, 0, DateTimeKind.Utc), new DateTime(2021, 6, 1, 0, 0, 0, DateTimeKind.Utc), BIP9DeploymentsParameters.DefaultMainnetThreshold)
+            };
+
+            consensusFactory.Protocol = new ConsensusProtocol()
+            {
+                ProtocolVersion = ProtocolVersion.FEEFILTER_VERSION,
+                MinProtocolVersion = ProtocolVersion.POS_PROTOCOL_VERSION,
+            };
+
+            Consensus = new NBitcoin.Consensus(
+                consensusFactory: consensusFactory,
+                consensusOptions: consensusOptions,
+                coinType: setup.CoinType,
+                hashGenesisBlock: genesisBlock.GetHash(),
+                subsidyHalvingInterval: 210000,
+                majorityEnforceBlockUpgrade: 750,
+                majorityRejectBlockOutdated: 950,
+                majorityWindow: 1000,
+                buriedDeployments: buriedDeployments,
+                bip9Deployments: new NoBIP9Deployments(),
+                bip34Hash: null,
+                minerConfirmationWindow: 2016, // nPowTargetTimespan / nPowTargetSpacing
+                maxReorgLength: 500,
+                defaultAssumeValid: null,
+                maxMoney: long.MaxValue,
+                coinbaseMaturity: 50,
+                premineHeight: 2,
+                premineReward: Money.Coins(setup.PremineReward),
+                proofOfWorkReward: Money.Coins(setup.PoWBlockReward),
+                targetTimespan: TimeSpan.FromSeconds(14 * 24 * 60 * 60), // two weeks
+                targetSpacing: setup.TargetSpacing,
+                powAllowMinDifficultyBlocks: false,
+                posNoRetargeting: false,
+                powNoRetargeting: false,
+                powLimit: new Target(new uint256("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")),
+                minimumChainWork: null,
+                isProofOfStake: true,
+                lastPowBlock: setup.LastPowBlock,
+                proofOfStakeLimit: new BigInteger(uint256.Parse("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false)),
+                proofOfStakeLimitV2: new BigInteger(uint256.Parse("000000000000ffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false)),
+                proofOfStakeReward: Money.Coins(setup.PoSBlockReward),
+                proofOfStakeTimestampMask: setup.ProofOfStakeTimestampMask
+            );
+
+            // HashKey Definition
+            byte[] newHashkey = Encoders.ASCII.DecodeData("Rutanio seed");
+            ExtKey.OverrideHashKey(newHashkey);
+
+            Consensus.PosEmptyCoinbase = RutanioSetup.Instance.IsPoSv3();
```

Comment:
If Rutanio is not completely bootstrapped I may recommend using V4 (and not use the trx time field anymore) but we need to discuss this in discord

Reply:
Let's talk in discord. Removing the trx time field to be like BTC field structure?